docs/table/index.rst

```diff
@@ -21,7 +21,8 @@ notable capabilities of this package are:
 * Create a new table by selecting rows or columns from a table.
 * Perform :ref:`table_operations` like database joins, concatenation, and binning.
 * Maintain a table index for fast retrieval of table items or ranges.
-* Manipulate multidimensional columns.
+* Manipulate multidimensional and `structured array
+  <https://numpy.org/doc/stable/user/basics.rec.html>` columns.
```

Comment:
You are missing underscore here, though I would recommend proper intersphinx instead of hardcoding URL, if possible.

```suggestion
  <https://numpy.org/doc/stable/user/basics.rec.html>`_ columns.
```

Reply:
Thanks, done! (I'll leave figuring out how to dot that with intersphinx for another time...)